src/api/ResponseHelper.js

```diff
@@ -38,11 +38,11 @@ const parseErrorArray = (errors) =>
  * @param {Object} data e.g. { data: { id: 1}}
  * @returns { Boolean, {Object}, String} { success, data, errors } e.g. { success: true, data: {id: 1}} or {success: false, errors: 'there was an error'}
```

Comment:
Looks like this might be out of date.

Reply:
OK. I will add it as a note for changing.